packages/gatsby/content/getting-started/migration.md

```diff
@@ -7,54 +7,47 @@ description: A step-by-step and in-depth migration guide from Yarn 1 (Classic) t
-12. Run `yarn plugin list` to see what other official plugins exist and might be useful
+11. Run [`yarn plugin import interactive-tools`](/cli/plugin/import) if you want [`upgrade-interactive`](/cli/upgrade-interactive)
+12. Run [`yarn plugin list`](/cli/plugin/list) to see what other official plugins exist and might be useful
 13. Commit the yarn plugins
 
-Now you have a working yarn v2 setup, but your repository might still need some extra care. Some things you'll need to change:
+Good, you should now have a working Yarn install! Some things might still require a bit of work (for instance we deprecated [arbitrary `pre/post`-scripts](/advanced/lifecycle-scripts)), but those special cases will be documented on a case-by-case basis in the rest of this document (for example [here](http://localhost:8000/getting-started/migration#explicitly-call-the-pre-and-post-scripts)).
 
-- There is no `node_modules` folder and no `.bin` folder. If you relied on these, [call `yarn run` instead](##call-binaries-using-yarn-run-rather-than-node_modulesbin).
-- Replace any calls to `node` that are not inside a Yarn script with `yarn node`
-- Custom pre-hooks (e.g. prestart) need to be called manually now
+## Switching to Plug'n'Play
 
-All of this and more is documented in the following sections. In general, we advise you at this point to try to run your application and see what breaks, then check here to find out tips on how to correct your install.
-
-## General Advices
-
-### Upgrade to Node 10+
+This step is completely optional - while we recommend to use Plug'n'Play for most new projects, it may sometimes require an average time investment to enable it on existing projects. For this reason, we prefer to list it here as a separate step that you can look into if you're curious or simply want the absolute best of what Yarn has to offer.
 
-Yarn doesn't support Node 8 anymore, as it's reached its end of life in December and won't receive any further update.
+### Before we start
 
-### Run the doctor
+Plug'n'Play enforces strict dependency rules. In particular, you'll hit problems if you (or your dependencies) rely on unlisted dependencies (the reasons for that are detailed in our [Rulebook](/advanced/rulebook)), but the gist is that it was the cause of many "project doesn't work on my computer" issues, both in Yarn and other package managers).
 
-Run `npx @yarnpkg/doctor .` (or `yarn dlx @yarnpkg/doctor .`) in your project to quickly get an overview of potential issues found in your codebase. For example here's what `webpack-dev-server` would reveal:
+To quickly detect which places may rely on unsafe patterns, run `npx @yarnpkg/doctor` (or `yarn dlx @yarnpkg/doctor`) in your project - it'll statically analyze your sources to try to locate all the most common issues that could lead in a subpar experience. For example here's what `webpack-dev-server` would reveal:
```

Comment:
Since at this point they already have v2 setup I think we should only suggest `yarn dlx`